debian/control

```diff
@@ -1,15 +1,15 @@
 Source: hw-probe
-Section: utils
 Priority: optional
+Section: utils
 Maintainer: Mikhail Novosyolov <mikhailnov@dumalogiya.ru>
 Build-Depends: debhelper (>=9), perl-base (>= 5), make
-Standards-Version: 4.0.0
+Standards-Version: 3.9.6
 Homepage: https://github.com/linuxhw/hw-probe
 
 Package: hw-probe
 Architecture: all
-Depends: perl-base (>= 5), curl, hwinfo, dmidecode, pciutils, usbutils, hdparm, smartmontools, edid-decode
-Recommends: systemd, cpuid, inxi, sysstat, memtester, rfkill, xinput, i2c-tools, pnputils, mesa-utils, vainfo, vulkan-utils, upower
-Suggests: acpica-tools, hplip, numactl
+Depends: perl-base (>= 5), curl, hwinfo, dmidecode, pciutils, usbutils, hdparm, smartmontools, util-linux
+Recommends: inxi, cpuid, upower, hdparm, sysstat, memtester, edid-decode, rfkill, xinput, i2c-tools, pnputils, acpica-tools, mesa-utils, vainfo, vulkan-utils, linuxinfo, numactl, mcelog, procps, iostat, lm-sensors
```

Comment:
1. Why we need linuxinfo?
2. sysstat already provides iostat
3. edid-decode is necessary to detect a monitor when hwinfo failed to detect it. Why it's just a recommended package?

Reply:
fixed by https://github.com/linuxhw/hw-probe/pull/13/commits/2eaab86e9ecfc62d32d90d932dedc4113838ffd9